### ISSUE TITLE
Does articulate capable of handling complex bot

### ISSUE THREAD

**Issue by Beherasaptami** [samtecspg/articulate#1051]
I have one question , i want my bot to handle complex things something like  bot will ask question like 
do you want to watch movies ? and buttons yes or no so if user will hit on yes  it will ask 4-5 more questions to user like name and where they want to watch and if user will hit on no button then it will ask some other queries 

is it possible to handle form actions and the above query using articulate

**Comment by wrathagom** [comment]
yes, you would use a combination of slots + modifiers + action chaining.

I can try to build up an example for you.

**Comment by Beherasaptami** [comment]
okay , thanks can you show one short example of the above scenario

**Comment by Beherasaptami** [comment]
I tried the above pattern but i have one doubt like i have multiple questions whose buttons will be yes or no , let me give you one example 
1. do you want to watch movies ?
  yes or no if user will hit yes button then it will redirect to question 2 ,if no it will give some random text .
2. you want to watch action movies ?
yes or no if yes then it will trigger question no 3
3. want to book tickets ?
yes  or no 
 let's take the above scenario ,suppose first action triggered after that i will choose yes button then there maybe possibility that bot got confuse like both both second and third question will trigger for yes option , how to tackle with this kind of scenario, any suggestions

**Comment by Beherasaptami** [comment]
any update like how can i tackle this
